enclave_manager/avalon_enclave_manager/sgx_work_order_request.py

```diff
@@ -32,15 +32,12 @@ def __init__(self, config, work_order, ext_data=""):
         if enclave_type == "KME":
             self.enclave = importlib.import_module(
                 "avalon_enclave_manager.kme.kme_enclave")
-            self.enclave_type = self.enclave.KME_ENCLAVE
         elif enclave_type == "WPE":
             self.enclave = importlib.import_module(
                 "avalon_enclave_manager.wpe.wpe_enclave")
-            self.enclave_type = self.enclave.WPE_ENCLAVE
         else:
             self.enclave = importlib.import_module(
                 "avalon_enclave_manager.singleton.singleton_enclave")
-            self.enclave_type = self.enclave.SINGLETON_ENCLAVE
```

Comment:
Instead of else can you use specific name for singleton like "pinned_enclave". In toml if specify any value then it will go to singleton. Validation team has bug filed for this.

Reply:
Makes sense. Will add a check for singleton enclave as well.